server/datastore/mysql/migrations/tables/20240717171504_AddGlobalStatToVulnHostCounts.go

```diff
@@ -0,0 +1,45 @@
+package tables
+
+import (
+	"database/sql"
+	"fmt"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20240717171504, Down_20240717171504)
+}
+
+func Up_20240717171504(tx *sql.Tx) error {
+	stmt := `
+	ALTER TABLE vulnerability_host_counts
+	ADD COLUMN global_stats tinyint(1) NOT NULL DEFAULT 0
```

Comment:
Don't we also need to move the current team_id=0 stats to global_stats=1 as part of migration?

Maybe `update table set global_stats=1 where team_id=0`?

Reply:
added